### ISSUE TITLE
Incorrect C_SIZE_MULT in sdcard driver

### ISSUE THREAD

**Issue by whaleygeek** [micropython/micropython-infineon#8]
The ```count()``` method returns the wrong number of sectors in this port.

This is because the wrong SIZE_MULT calculation constant is used in the ```init_card()``` function

https://github.com/micropython/micropython-infineon/blob/master/drivers/sdcard/sdcard.py#L81

```self.sectors = ((csd[8] << 8 | csd[9]) + 1) * 2014```

The V2 CSD structure returns the card size in 512KB chunks.

The normal calculation as per the sdcard.org specification (SD Specifications, Part 1, Physical Layer Simplified Specification Version 2.00 September 25, 2006) is:

```
nbytes = ((CSIZE-1) * 512 * 1024)  # V2 cards always report in 512KB chunks
nblocks = nbytes / 512  # V2 cards always work with a 512 byte sector/BLOCK_LEN
```

This reduces to:

```
nblocks = (CSIZE-1) * 512 * 1024 / 512
nblocks = (CSIZE-1) * 1024
```

I guess this is just a typo by the original author, as you have the same digits but transposed.

cc @dpgeorge @peterhinch

**Comment by dpgeorge** [comment]
This repository is no longer maintained.  The SD card driver has moved to `micropython-lib`, at https://github.com/micropython/micropython-lib/blob/master/micropython/drivers/storage/sdcard/sdcard.py.  And this bug was fixed some time ago.

@whaleygeek are you using this repo (eg on an Infineon part)?

**Comment by whaleygeek** [comment]
@dpgeorge Thanks for the redirect. I only landed on this page because I googled for something about SDCards, I'll use the new lib link now thanks.

Just FYI, Github supports archived projects, which prevents them looking like an active project, and prevents issues being logged. It is also possible to put a note in the README on an old project indicating that it is no longer active, and where to go instead, if either or both of those would be a good way to prevent others falling into the same trap as me.

Thanks for your speedy response.
